src/main/scala/SBTS3Resolver.scala

```diff
@@ -129,12 +131,15 @@ object SbtS3Resolver extends AutoPlugin {
 
   // Default settings
   override def projectSettings: Seq[Setting[_]] = Seq(
-    awsProfile := "default",
+    awsProfile := None,
     s3credentials :=
-      new ProfileCredentialsProvider(awsProfile.value) |
+      new ProfileCredentialsProvider(awsProfile.value.orNull) |
       new EnvironmentVariableCredentialsProvider() |
       InstanceProfileCredentialsProvider.getInstance(),
```

Comment:
I'd like to change `s3credentials` default similarly to the new default for `s3region`. I think consistency here is more important than preserving compatibility:
- the default will be mentioned in the readme
- the change will be mentioned in the release notes
- I think it won't affect most people, because usually you either rely on the default credentials chain, or set credentials explicitly. Also if you were using `awsProfile` before, then you also shouldn't notice the change.

So change this, please, in the same way as the `s3region`.